protocol/directory.go

```diff
@@ -31,15 +36,18 @@ func InitDirectory(epDeadline merkletree.TimeStamp, vrfKey vrf.PrivateKey,
 	return d
 }
 
-func (d *ConiksDirectory) Update() {
+func (d *ConiksDirectory) Update(up *UpdatePolicies) {
```

Comment:
> I can't find a pointer to it though. Maybe it was only on hangout. We should get better at documenting these things that diverge from the paper.

Same here, I looked, but couldn't find any.

>  We should get better at documenting these things that diverge from the paper.

Absolutely!

> I remember that we agreed to let client compute the timestamp for themselves (on hangout).

Ah, I couldn't remember, sorry :(

>  If EpochDeadline is an actual time stamp, I think there has not much advantages when using Epoch as counter ?

We should discuss this in a separate issue as Arlo said.